### PR TITLE
[Feature] Risk calculation in store

### PR DESCRIPTION
**Pull request by aleksandr-tikhonov**

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Services/Risk/Calculation/RiskCalculation.swift [hunk 1]
```diff
@@ -77,10 +77,10 @@ enum RiskCalculation {
 		}
 
 		// Precondition 1 - Exposure Notifications must be turned on
-		guard preconditions.isGood else {
-			// This overrides all other levels
-			return .success(.inactive)
-		}
+//		guard preconditions.isGood else {
```
Comment (aleksandr-tikhonov) [inline src/xcode/ENA/ENA/Source/Services/Risk/Calculation/RiskCalculation.swift:80]: @inf2381 should we comment code here?

Comment (inf2381) [reply src/xcode/ENA/ENA/Source/Services/Risk/Calculation/RiskCalculation.swift:80]: @aleksandr-tikhonov I think we should revert these changes and fix the root cause in a separate PR

**Review by inf2381 — APPROVED** [review]

**Comment by inf2381** [comment]
@haosap Should we merge this PR or not?

**Comment by haosap** [comment]
Hi @inf2381 ,
Please don't do it.
I put it on hold.  The risk calc is done in real time instead of storing it.

**Comment by i336616** [comment]
> Hi @inf2381 ,
> Please don't do it.
> I put it on hold. The risk calc is done in real time instead of storing it.

If this is to close https://jira.itc.sap.com/browse/EXPOSUREAPP-484, we persist the previousSummary & previousSummaryDate instead (ENExposureDetectionSummaryContainer), covered by this PR: https://github.com/corona-warn-app/cwa-app-ios/pull/293

**Comment by haosap** [comment]
Hi @ChristianKienle ,
Shall we close it now?

**Pull request closed without merging**